Code/Source/sv4gui/Modules/ImageProcessing/sv4gui_ImageProcessingUtils.cxx

```diff
@@ -0,0 +1,524 @@
+#include "sv4gui_ImageProcessingUtils.h"
+
+
+#include <itkVTKImageToImageFilter.h>
+#include <itkThresholdImageFilter.h>
+#include <itkCastImageFilter.h>
+
+#include <itkImageToVTKImageFilter.h>
+#include <itkVTKImageToImageFilter.h>
+
+#include <vtkMarchingCubes.h>
```

Comment:
Separate and group itk and vtk headers if we can. We should try to bunch by library.